Automattic/jetpack
# License Activation Notice: fix issue in license activation notice that appears even after being dismissed

Fixes 1202619025189113-as-1202655518335358

#### Changes proposed in this Pull Request:

It fixes an issue in the user license activation notice banner that appears even after being dismissed.

#### Other information:


#### Does this pull request change what data or activity we track or use?

**No**

#### Testing instructions:

1. Set up a new test site using JN with this branch.
2. Visit the newly created site -> Click on **Jetpack** nav item -> Activate Jetpack.
3. Go to https://jetpack.com/ and buy any product and skip the site selection.
4. Now, go to your site -> Jetpack(refresh if you are on the same page), and you should see the activation notice with the message `Activate <PRODUCT_NAME> ...`
5. Now dismiss the notice by clicking on the `X` icon on the notice and verify that the notice doesn't appear again

Example notice

<img width="1047" alt="Screenshot 2022-07-26 at 5 18 54 PM" src="https://user-images.githubusercontent.com/10586875/181000460-8ce85aa9-009c-408e-8490-14371afa2aac.png">


> **_NOTE:_**  This issue is caused by merging [Show product name on activation notice](https://github.com/Automattic/jetpack/pull/23409.) The expected behavior is to hide the notice banner when a user activates a license until a new license is issued; check [Display notice when a user has unactivated product license keys](https://github.com/Automattic/jetpack/pull/21474) for more details.
